### DISCUSSION TITLE
[Request]. Add YTNoPaidPromo

### DISCUSSION THREAD

**Discussion by khangbinhdl** [qnblackcat/uYouPlus#261] [Ideas]
Hey bro, can you add this tweak to UYouPlus thanks.
https://github.com/PoomSmart/YTNoPaidPromo

**Comment by qnblackcat** [comment]
the tweak has been added already (https://github.com/qnblackcat/uYouPlus/commit/c2d1581698f0b9363e47aef79a71c3c5b1786f05)
